src/createServer.js

```diff
@@ -1,3 +1,78 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function handleErrors(toCase, text) {
+  const errors = [];
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  if (!toCase) {
+    errors.push({
+      message: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!text) {
+    errors.push({
+      message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!cases.includes(toCase)) {
+    errors.push({
+      message: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+    });
+  }
+
+  return errors;
+}
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('content-type', 'application/json');
+
+    const [url, queryString] = req.url.split('?');
+    const params = new URLSearchParams(queryString || '');
+
+    const text = url.replace(/^\//, '');
+    const toCase = params.get('toCase');
+
+    const CODE_STATUS_400 = 'Bad Request';
+    const CODE_STATUS_500 = 'Internal Server Error';
+
+    const errors = handleErrors(toCase, text);
+
+    if (errors.length > 0) {
+      res.writeHead(400, CODE_STATUS_400);
```

Comment:
you can write all status codes `200,400,500` in constants.

Reply:
I also write about a number which you use in this case)
 res.writeHead(`400`, CODE_STATUS_400);